neutron-org/wasmd
# sdk 46

O- Bump sdk version to v0.45.5
- Bump github.com/stretchr/testify from 1.7.2 to 1.8.0
- Remove unnecessary string
- Prevent migration to a restricted code
- Restrict code access config modifications
- Remove outdated proto.md
- Remove unnecessary error check
- Create .gitpod.yml
- Don't pass non-wasm events to reply()
- Oops
- Add changelog for fix
- Fix tests in keeper
- Add sorting fix from terra devs
- Fix imports
- Code cleanup suggested by @peterbourgon
- put sorting fix next to event filtering
- Add changelog entry
- 457
- upgrade ci
- go get -> go install
- change denom to new format
- Update config.yml
- Revert "change denom to new format"
- bump ibc-go to v3.2.x
- Update config.yml
- but not done
- TestInitGenesis fails
- use gofumpt
- some minor fix
- Update config.yml
- linting
- Setifexists
- fix wasm export
- SetInterfaceRegistry
- Update dependabot.yml
- fix gas limit
- nil srcExp
- govv1
- govv1 in TestQueryContractInfo
- snapshot test
- fix gas in test, v1beta -> v1
- govv1 in encoding test
- fix wasm.go
- with valset in bench
- Make app keepers public (#951)
- Fix keeper test
- Bump actions/checkout from 2.3.5 to 3.0.2
- depracte-ioutil (#934)
- Bump github/codeql-action from 1 to 2 (#954)
- Bump bufbuild/buf-setup-action from 0.7.0 to 1.7.0
- close 815: upgrade to go 1.18 (#866)
- Revert changes in protogen script (#956)
- sdk v0.46.1 and tm v0.34.21
- Ugrade sdk to v0.45.7 (#959)
- Bump github.com/prometheus/client_golang from 1.12.2 to 1.13.0
- Fix codeql analyzer branch config
- automatic fixes from "golangci-lint run ./... --fix"
- bump ica
- fix proposal integration test: line 178 shoud be require.Error
- sdk v0.45.8
- Update CHANGELOG.md
- ibc 5-rc1
- Charge gas to unzip wasm code (#898)
- Remove obsolete cli gov params
- add params query
- Upgrade thirdparty protos (#977)
- add pin code flag to store code proposal
- change default behavior to always pin
- fix tag
- turbo 46
- turbo46
- Bump github.com/spf13/viper from 1.12.0 to 1.13.0
- Upgrade to wasmvm v1.1.0
- Update changelog
- Update README.md
- Introduce AccessTypeAnyOfAddresses
- Prevent empty addresses list
- Formatting only
- Parse any of address in CLI; more tests
- More tests
- Upgrade keyring/ go-keychain dependency
- ibc-go v3.2.0 (#937)
- Add query for the total supply of a coin (#903)
- fix bug of contract history
- Test contract history order
- Revert "Create .gitpod.yml"
- Better wasmvm 1.1 integration test (#988)
- Rename features to availableCapabilities (#993)
- Link to medium  article
- Make contract addresses predictable
- Revert "turbo 46"
- Revert "turbo46"
- fix flagproposaltype
- update
- use keyed fields
- Add factory contract scenario
- Fix genesis message tooling
- Minor cleanup
- Bump ibc proto to v3.2.0
- bank keeper
- fix test genesis with valset
- gov address
- gas in test
- Update changelog
- Add migration note
- Fix master -> main in CI config
- Always run docker-image
- Add libwasmvm version check
- Bump wasmvm to 1.1.1
- Upgrade .a files to 1.1.1
- Fix path to proofs in protos
- Bump github.com/cosmos/ibc-go/v3 from 3.2.0 to 3.3.0
- Prune vesting accounts balances (#1003)
- Implement improvements to new address generation (#1014)
- Changelog for v0.29.0-rc1 (#1018)
- ibc-go v5.0.0
- update gas per suggestion
- bring branch up to date with ibc-go v5.0.0
- attempt to fix remaining bug
- Fix: typos
